src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -142,6 +142,7 @@ bool GetForInnerPathExistence(KeyValuePair<string, JsonElement> o) =>
             Eip6780TransitionTimestamp = chainSpecJson.Params.Eip6780TransitionTimestamp,
             Rip7212TransitionTimestamp = chainSpecJson.Params.Rip7212TransitionTimestamp,
             OpGraniteTransitionTimestamp = chainSpecJson.Params.OpGraniteTransitionTimestamp,
+            OpHoloceneTransitionTimestamp = chainSpecJson.Params.OpHoloceneTransitionTimestamp,
```

Comment:
The first issue was that we missed one of these copies which resulted in a `IReleaseSpec` where Holocene was never enabled. I think some parts of the system were working correctly since we were getting this timestamp from a different source.

Either way, this kind of work is repetitive and error prone and it would be great if we could come up with a better solution that does not require such manual work.

Reply:
Let's remove OpHoloceneTransitionTimestamp from ChainSpecLoader. We have OptimsimChainSpecEngineParameters for that. Also we should remove OpGraniteTransitionTimestamp, but not in this pr proabably

Reply:
Alright, that sounds good, but I'm having a bit of issues getting an instance of `OptimismChainSpecEngineParameters` in certain places like in `OptimismPayloadAttributes.Validate`:

https://github.com/NethermindEth/nethermind/blob/fbf855bc468fc3a59a529ee4685f48942fb013c8/src/Nethermind/Nethermind.Optimism/Rpc/OptimismPayloadAttributes.cs#L117-L127